server/src/test/java/com/google/fhir/gateway/BearerAuthorizationInterceptorTest.java

```diff
@@ -323,11 +323,30 @@ public void authorizeRequestMetadata() throws IOException {
         equalTo("OAuth"));
   }
 
+  @Test
+  public void authorizeAllowedUnAuthenticatedRequest() throws IOException {
```

Comment:
nit: I think "unauthenticated" is one word and should be CamelCased as `...AllowedUnauthenticated...`.

Reply:
I have also used this for config properties. Updated those also use unauthenticated as a single word.